lib/semian.rb

```diff
@@ -296,7 +296,19 @@ def require_keys!(required = [], **options)
   end
 end
 
+def hostname
+  v = %w(KUBE_HOSTNAME KUBE_HOST_NAME KUBE_NODENAME KUBE_NODE_NAME NODENAME NODE_NAME HOSTNAME HOST_NAME)
```

Comment:
small optimization: Looks like `KUBE_HOSTNAME` is the thing for shopify core. Outside of that, I've typically seen `HOSTNAME`. Perhaps make `HOSTNAME` the second entry in this list?

Reply:
I wanted to leave things that are commonly defined to the end of the list, to allow the custom ones to be selected.

```
root@web-5b44f8fbbd-27bh5:/app# set | grep -i -e HOST -e NODE
HOSTNAME=web-5b44f8fbbd-27bh5
HOSTTYPE=x86_64
HOST_IP=10.192.1.141
KUBERNETES_SERVICE_HOST=10.188.0.1
KUBE_HOSTNAME=gke-tier1-us-east-4-default-2-c7098a55-z3jr
PROXYSQL_ADMIN_MASTER_SERVICE_HOST=10.188.5.35
PROXYSQL_MASTER_SERVICE_HOST=10.188.11.180
SMTP_SERVICE_HOST=smtp.smtp-gke-production
WEB_PERF_SERVICE_HOST=10.188.11.159
WEB_SERVICE_HOST=10.188.15.146
```

Notice that `HOSTNAME=web-5b44f8fbbd-27bh5` but `KUBE_HOSTNAME=gke-tier1-us-east-4-default-2-c7098a55-z3jr` is what we're really after.

Reply:
Sounds good :+1: